### ISSUE TITLE
Docker container

### ISSUE THREAD

**Issue by Vysp3r** [BeamMP/BeamMP-Server#302]
I think it could be really nice if we could have a Docker image of BeamMP so that we can easily spin servers.
A Docker compose file would also be a nice idea.
I personally like to run my stuff in Docker, because I can manage them easily with Portainer.

**Comment by lionkor** [comment]
Hi, great idea! Its trivial to dockerize BeamMP servers, but I'd be happy to add some docker compose and dockerfile examples to this repo :)

**Comment by OfficialLambdax** [comment]
Imma juts put this here as thats what i use https://github.com/OfficialLambdax/BeamMP-ServerDocker

**Comment by Vysp3r** [comment]
@OfficialLambdax I think it would be better for them to manage it, but thank you I'll use that in the meantime!

**Comment by OfficialLambdax** [comment]
@Vysp3r yeah correct, that one is and will stay unoffical, i shouldnt create the official docker project with my limited containering knowledge.

**Comment by carsakiller** [comment]
I've been using [RouHim's container](https://github.com/RouHim/beammp-container-image) for years now and have no complaints!

**Comment by enzofrnt** [comment]
Here is it :
https://github.com/BeamMP/BeamMP-Server/pull/459
